### ISSUE TITLE
Except on get_info!

### ISSUE THREAD

**Issue by Velarde510** [instabot-py/instabot.py#1131]
> Fill in all requirements *before* submitting an issue.
> Do not delete any of the template, fill all of it in; even if you think it doesn't apply to your issue.
> If you fail to follow these simple instructions, we will close the ticket.

- [x] This is a checked box. **Do not leave spaces around the `x`!**

- [x] I have searched for other issues with the same problem.

## Hosting
- [ ] Are you hosting the bot on a VPS or other Cloud hosting?
- [x] If so; have you tried running the code on your local machine?

#### Select one:
- [ ] Bug
- [ ] Feature Request
- [x] Technical Help
- [ ] Other

### Operating System? (include version)
 Windows 10

## Python Version?
3

## Description

04.04.2018_19:00 Checking user info..
04.04.2018_19:00 Except on get_info!
04.04.2018_19:00 "account" : Get user info

x100

Help me pls!

```

**Comment by uzico** [comment]
Happens to me as well, only methon 0 works  - all others give this error

**Comment by spodZ** [comment]
Can you send me your source files? Mines keep getting this errors and the developers are busy

**Comment by darynk** [comment]
Same here

**Comment by alyysaafan** [comment]
because this "https://www.instagram.com/?__a=1" is deprecated as it returns json "{"graphql":null}"
which means it will show exceptions to get users info and for unfollowing also .

# so please any of the developers here can update the bot

**Comment by Dalrabadi** [comment]
Yeah I am having the same problem when I debugged I reached the same conclusion as alyysaafan. Unfortunately, I don't know much about API's I know that in order to get the json info you append /?__a=1a and I tried that but it is not working. I am not sure what the new API would be

**Comment by ChipWolf** [comment]
I'll try and replicate, if anyone has a solution, let me know.

**Comment by robneir** [comment]
This is an issue I am getting as well. I added a print statement to print this line 
"15.09.2018_11:52 {"graphql":null}" to see what the json data looked like that I was getting back. Looks like graphql is null obviously. Not sure why. Please help with this. I also pasted my code in my recent_feed.py which is where this problem is occurring.

**OUTPUT:**

15.09.2018_11:51 Except on get media rob!!
xxxxxxx user info list is empty!!! xxxxxxxxx
15.09.2018_11:52 violavoi : Get media id on recent feed
 15.09.2018 11:52
15.09.2018_11:52 {"graphql":null}
15.09.2018_11:52 Except on get media rob!!
15.09.2018_11:52 ======> Get media id by Tag <======
15.09.2018_11:52 Get media by user success!
15.09.2018_11:52 Current Index = 39 of 62 medias
15.09.2018_11:52 Current Index = 31 of 61 medias

**RECENT_FEED.PY:**

#!/usr/bin/env python
# -*- coding: utf-8 -*-
import datetime
import json
import time


def get_media_id_recent_feed(self):
    if self.login_status:
        now_time = datetime.datetime.now()
        log_string = "%s : Get media id on recent feed \n %s" % (
            self.user_login, now_time.strftime("%d.%m.%Y %H:%M"))
        self.write_log(log_string)
        url = 'https://www.instagram.com/?__a=1'
        try:
            r = self.s.get(url)
            all_data = json.loads(r.text)
            self.write_log(r.text)
            self.media_on_feed = list(all_data['graphql']['user']['edge_web_feed_timeline']['edges'])
            log_string = "Media in recent feed = %i" % (
                len(self.media_on_feed))
            self.write_log(log_string)
        except:
            self.media_on_feed = []
            self.write_log('Except on get media rob!!')
            time.sleep(20)
            return 0
    else:
        return 0
